documentation/docs/deploy/flags.md

```diff
@@ -28,6 +28,7 @@ Parameters/Flags can be defined in 2 ways:
 |debug|Boolean|Enables or disables starting processes in debug mode. It overrides any debug setting in configuration. If not set debugging is enabled.|
```

Comment:
configuration.md  has to be updated

Reply:
A blog post has to be added, to explain it from more high level picture. Look at example here: https://xebialabs.github.io/integration-server-gradle-plugin/blog/2021/12/10/run-deploy-openshift-cluster-with-operator